extensions/wikia/RobotsTxt/tests/WikiaRobotsTest.php

```diff
@@ -100,12 +115,9 @@ private function isSpecialPageAllowed( RobotsTxtMock $robotsTxtSpy, $page ) {
 	 */
 	public function testNonProductionEnvironment( $env ) {
 		$this->mockGlobalVariable( 'wgWikiaEnvironment', $env );
+		$this->mockGlobalVariable( 'wgRobotsTxtBlockedWiki', false );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wgRobotsTxtBlockedWiki" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)